src/main/java/com/baulsupp/oksocial/Main.java

```diff
@@ -501,6 +514,38 @@ public void initialise() throws Exception {
         new Authorisation(serviceInterceptor, credentialsStore, authClient, outputHandler);
 
     clientBuilder.networkInterceptors().add(0, serviceInterceptor);
+
+    if (zipkin) {
+      OkHttpSender sender = OkHttpSender.create("http://localhost:9411/api/v1/spans");
+      AsyncReporter<zipkin.Span> reporter = AsyncReporter.create(sender);
+
+      Tracing tracing = Tracing.newBuilder()
+          .localServiceName("oksocial")
+          .reporter(reporter)
+          .sampler(Sampler.ALWAYS_SAMPLE)
+          .build();
+
+      HttpTracing httpTracing = HttpTracing.create(tracing);
+
+      Tracer tracer = tracing.tracer();
+
+      Consumer<TraceContext> opener =
+          tc -> closeables.add(
+              () -> openLink("http://localhost:9411/zipkin/traces/" + tc.traceIdString()));
```

Comment:
if the goal is to see the trace, you could hit /api/v1/trace/+id, maybe using something to pretty-print it